package.json

```diff
@@ -33,12 +32,13 @@
   },
   "dependencies": {
     "@heroicons/react": "^2.1.1",
+    "@netlify/plugin-nextjs": "^5.2.2",
```

Comment:
@sherakama We can remove this from package.json now right 👀 ? We can do that in the release PR

Reply:
I left it in as we did run into a conflict with this plugin and our vault plugin so I was cautious about locking the versions. When we remove it, Netlify will keep it up to date for us but we run the risk of it introducing a build error without us changing anything.

Reply:
I can go either way. Build errors won't cause the site to go down but it may delay content getting published and require us to scramble to figure out whats going on in the future. The trade, is that we will have to periodically update the plugin as well.

Reply:
Oh I didn't realize - sorry I removed it in the release PR. I can put it back in 😄

Reply:
It is all a trade and I can live with either scenario. You're also on the call sheet if this site starts to have a sad so think about which option you'd like to go with.

Reply:
That makes sense - I'll put it back in my horizontal changemaker card PR, then it can go into the release for that work since we'll probably be ok for now 🤞🏼 Thanks @sherakama !